docs/portal/applications/updating-an-application.md

```diff
@@ -1,24 +1,66 @@
-# Updating an Application
+# Quick Updating an Application
 
 The following procedure describes how to update applications from the Applications page. You can also update applications from the form used to [view application information](</docs/portal/applications/viewing-application-information.md>).
 
 1. In the left pane, click **Applications**.
-2. Find the application you want to update, then click its **Actions** menu and select **Update**. An Update Application form similar to the following appears.
+2. Click the name of the application you want to update.
+3. At the top right of the form, click the **Update** button, and then select one of the following quick update options:<ul><li> **Redistribute** allows you to add, edit, and delete distribution targets.</ul></li><ul><li>**Set container image** allows you to update your pods to a newer application version.br></ul></li><ul><li>**Upgrade resources** allows you to upgrade the spec of your container instance to maintain performance and/or save costs.</ul></li>
 
 ![null](</docs/resources/images/applications/applications-update.png>)
 
-3. Complete the fields in the Update Application form. Required fields are denoted by an asterisk (\*).
+##Redistributing Your Application
```

Comment:
Many headings on this page are not in correct format. Please check.

Reply:
I confirmed that the heading formats are correct:
Heading 1 = Updating an Application
Heading 2 = Redistributing Your Application
Heading 3 = Applications Distributed per Region
Heading 3 = Applications Distributed per Pop
Heading 2 = Setting the Container
Heading 2 = Upgrading Resources

Reply:
The preview of this MD file doesn't look correct to me. Can you check the preview on GitHub as well? https://github.com/mileweb/docs-content-ecp/blob/marc-dev/docs/portal/applications/updating-an-application.md

On my Chrome browser, only "Updating an Application" is correctly rendered. The "#" characters on the other headings are not interpreted. The last 2 headings are not even displayed in their own lines. Besides, please replace Pop with PoP in the heading "Applications Distributed per Pop", and please change the heading "Setting the Container" to "Setting Container Image".
![image](https://user-images.githubusercontent.com/43203744/90620639-4c5cea80-e245-11ea-9b89-113e89ded644.png)
![image](https://user-images.githubusercontent.com/43203744/90620779-7f9f7980-e245-11ea-90f0-825b9b6166ce.png)

Reply:
Fixed.